lib/paper_trail/compatibility.rb

```diff
@@ -18,7 +18,7 @@ module PaperTrail
   # versions.
   module Compatibility
     ACTIVERECORD_GTE = ">= 5.2" # enforced in gemspec
-    ACTIVERECORD_LT = "< 7.0" # not enforced in gemspec
+    ACTIVERECORD_LT = "< 8.0" # not enforced in gemspec
```

Comment:
Please specify `< 7.1`

Reply:
Could you explain why ? When the latest rails was 6.1 this was `<7.0` and not `< 6.2`, I believe as long as we are in the same major version there should be no breaking change ?

Reply:
Yeah, I can see why the previous value would be confusing. The previous value should have been `< 6.2`. The intention of this `Compatibility` range is for versions that we test against. IOW, if it's in the range, we are actively testing against it in CI.

Reply:
Got it !